integration_tests/dbt_project.yml

```diff
@@ -1,30 +1,32 @@
 name: 'marketo_integration_tests'
-version: '0.7.0'
+version: '0.8.0'
 profile: 'integration_tests'
 config-version: 2
 
 vars:
-
-    marketo__enable_programs: True
-    marketo__enable_campaigns: True
-
+    marketo__enable_campaigns:   true      # Un-comment line to enable testing the campaigns table
+    marketo__enable_programs:    true      # Un-comment line to enable testing the programs table
```

Comment:
Similar note from the source package. Can you add these to the config.yml to test them on/off